app/models/audit_activity/comment/add_comment.rb

```diff
@@ -0,0 +1,15 @@
+class AuditActivity::Comment::AddComment < AuditActivity::Base
+  def self.build_metadata(body)
+    {
+      comment_text: body
+    }
+  end
+
+  def title(*)
+    "Add comment"
+  end
+
+  def subtitle_slug
+    "Added"
```

Comment:
```suggestion
    "Comment added"
```
To match existing behaviour.